docs/advanced_topics/images/renditions.md

```diff
@@ -38,42 +38,33 @@ See also: [](image_tag)
 
 ## Prefetching image renditions
 
-```{versionadded} 3.0
-This following guidance is only applicable in Wagtail versions 3.0 and above.
+```{versionadded} 4.0
+This following guidance is only applicable in Wagtail versions 4.0 and above.
 ```
 
-When using a queryset to render a list of objects with images, you can make use of Django's built-in `prefetch_related()` queryset method to prefetch the renditions needed for rendering with a single additional query. For long lists of items, or where multiple renditions are used for each item, this can provide a significant boost to performance.
+When using a queryset to render a list of images, you can make use of Wagtail's built-in ``prefetch_renditions`` queryset method to prefetch the renditions needed for rendering with a single additional query. For long lists of items, or where multiple renditions are used for each item, this can provide a significant boost to performance.
```

Comment:
I think this guidance and example doesn't totally cover the same as the old example - we should try and have both kinds, querying on images directly (where you can use `prefetch_renditions`) and querying on a model that's simply related to an image (in Andy's example) where you can't, and `prefetch_related` becomes useful.